src/components/styles.js

```diff
@@ -1,5 +1,8 @@
 export default {
   colors: {
     primary: '#89AEE1',
```

Comment:
can primary be more descriptive of the actual color like the other colors are?

Reply:
👍